tests/integration/cmd_logs_test.go

```diff
@@ -54,6 +55,29 @@ var _ = Describe("odo logs command tests", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
+	When("in a devfile directory", func() {
+		BeforeEach(func() {
+			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
+			helper.Cmd("odo", "init", "--name", componentName, "--devfile-path", helper.GetExamplePath("source", "devfiles", "nodejs", "devfile.yaml")).ShouldPass()
+		})
+		When("not connected to any cluster or podman", Label(helper.LabelNoCluster), func() {
+			It("odo logs should fail with an error message", func() {
+				cmd := helper.Cmd("odo", "logs")
+				stderr := cmd.ShouldFail().Err()
+				Expect(stderr).To(ContainSubstring("unable to access the cluster"))
+			})
+
+			It("odo logs --platform podman should fail with an error message", func() {
```

Comment:
This spec should have the `podman` label no ?

Reply:
No, because we want to test it without podman

Reply:
Ah okay, I see. Part of the confusion was seeing "logs --platform podman".
But what you said also makes sense.